README.md

```diff
@@ -35,18 +32,7 @@ python setup.py install
 ```
 --->
 
-## To install manually - UNIX OS
```

Comment:
this is pretty much the same thing as the previous section. it's better to explain to download it as a zip file or checkout the repository with git in the above example, then have the same commands in one place for either situation as opposed to separating it like this